src/rules.scala

```diff
@@ -533,6 +533,19 @@ object Plugin extends sbt.Plugin with PluginFail {
     dexMaxHeap               := "1024m",
     dexMaxProcessCount       := java.lang.Runtime.getRuntime.availableProcessors,
     dexMulti                 := false,
+    dexMainClassesRules      := Seq(
+      "-dontobfuscate",
+      "-dontoptimize",
+      "-dontpreverify",
+      "-dontwarn **",
+      "-dontnote **",
+      "-forceprocessing",
+      "-keep public class * extends android.app.backup.BackupAgent { <init>(); }",
+      "-keep public class * extends java.lang.annotation.Annotation { *; }",
+      "-keep class android.support.multidex.** { *; }",
+      "-keep class com.android.tools.fd.** { *; }",
+      "-dontnote com.android.tools.fd.**,android.support.multidex.MultiDexExtractor"
```

Comment:
drop the 2 com.android.tools.fd.** lines

verify whether or not everything is correct without any -keep android.support.multidex at all